docs/inventory.md

```diff
@@ -255,6 +255,35 @@ If you need VMs with static UUIDs to allow them to be reused then the UUID for a
 
 By default an SSH key will be generated by the `deploy_cluster.yml` playbook. This can be disabled by adding `generate_ssh_keys = False` to the inventory. It is possible to configure the task generating the SSH key (see the docs for `community.crypto.openssh_keypair`) by setting `openssh_keypair_args` with a dictionary.
 
+### DNS and DHCP
+
+If you to only point to the dns configure by crucible you should also provide `upstream_dns` which points to another dns server which can provide records for non-crucible queries.
+
+You can control the ip addresses which dnsmasq listens to using the `listen_addresses` by default this will include both `120.0.0.1` and ansible's default IPv4 address for the host (`ansible_default_ipv4.address`). You may also configure the interfaces which dnsmasq responds by defining `listening_interfaces` as a list of the interfaces you which to listen to. 
```

Comment:
If I want to define the listening addresses, should I use `listen_addresses` or `listening_interfaces` in the inventory?
Maybe it is worth adding an example for both of them

Reply:
Added example